docs/yarn_classic.md

```diff
@@ -0,0 +1,159 @@
+# Yarn Classic (Yarn v1)
+
+<https://classic.yarnpkg.com/>
+
+* [Overview]
+  * [Supported dependencies types]
+  * [Dealing with Yarn Zero-Installs]
+ * [Fetching dependencies for Yarn Classic projects]
+ * [Handling of yarn-specific config files]
+ * [Setting up offline mirror]
+
+This document outlines the differences between Yarn Classic and Yarn v3 support.
```

Comment:
I would use "Yarn Berry" or just "Yarn" instead of Yarn v3 in the document. When we add support to Yarn v4, which is still Yarn Berry IIRC, this would not be up to date, unless Yarn v4 is completely different PM, which would require a separate documentation and CLI option. But I am not an expert on all yarn versions :smile:

Reply:
I already use `Yarn` and `Yarn Classic` interchangeably. I think it is safe to risk this ambiguity.